scripts/contacts/migrateContactsV1toV2.js

```diff
@@ -75,7 +80,7 @@ const migrateContactV1toV2 = (contact, accountId, contactAccountId) => {
       createdByApp,
       createdByAppVersion: undefined,
       doctypeVersion: DOCTYPE_CONTACTS_VERSION,
-      sourceAccount,
+      sourceAccount: undefined,
```

Comment:
Nit: we use `undefined` here and `null` a bit further down, I think using `null` everywhere would make more sense?